build.sbt

```diff
@@ -21,7 +63,10 @@ lazy val microservice = Project(appName, file("."))
     // ***************
   )
   .settings(publishingSettings: _*)
-  .configs(IntegrationTest)
+  .configs(IntegrationTest extend Test)
   .settings(integrationTestSettings(): _*)
   .settings(resolvers += Resolver.jcenterRepo)
-  .settings(CodeCoverageSettings.settings: _*)
+  .settings(PlayKeys.playDefaultPort := 9377)
```

Comment:
Just to confirm, is this port unique to `income-tax-state-benefits`?

Reply:
Yes